<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobMaxScriptSettingsWidgetUI</class>
 <widget class="QGroupBox" name="JobMaxScriptSettingsWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>MaxScript Settings</string>
  </property>
  <layout class="QVBoxLayout" name="mVerticalLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QCheckBox" name="mSilentCheck">
     <property name="text">
      <string>Max Silent Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m64BitCheck">
     <property name="text">
      <string>Run 64 Bit 3dsmax</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>222</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
